src/sections/General/Navigation/index.js

```diff
@@ -258,7 +258,14 @@ const Navigation = ({ theme, themeSetter }) => {
                   <div>
                     <ul>
                       <li className="mobile-nav-item">
-                        <Link to="/cloud-native-management/meshmap" className="menu-item" activeClassName="nav-link-active">MeshMap</Link>
+                        <Link to="https://meshery.layer5.io/registration" className="menu-item" activeClassName="nav-link-active">Get Started</Link>
```

Comment:
`<Link>` is for site internal links only. We'll need to switch to `<a>` for an external link.

Reply:
I had the same doubt, however a quick google search showed that React Router Links can also be used for external links. And it works well.

Do you still want me to change it to href?